src/core/types/shared.d.ts

```diff
@@ -0,0 +1,10 @@
+export type MessageObject = {
+	messageType: string,
+	message: string
+};
+
+export type MessageObjectArray = MessageObject[];
```

Comment:
This doesn't look good, it might just be worth changing references to `MessageObjectArray` as the value here

Reply:
Completed in https://github.com/4x8Matrix/OutSi-TS-Plugin/pull/1/commits/38b6b8f9db45b8ca9c4774339038aee60a72a11b.